src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinLivingEntity.java

```diff
@@ -182,6 +190,38 @@ private void hookAirJump(CallbackInfo callbackInfo) {
         }
     }
 
+    private ClientPlayerEntity player() {
+        return MinecraftClient.getInstance().player;
+    }
```

Comment:
You are already working with the client player entity instance inside the [MixinLivingEntity]. You can therefore use `this` instead.